cmd/create/cluster/cmd.go

```diff
@@ -1519,13 +1521,13 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 
 	// Custom tags for AWS resources
-	tags := args.tags
+	_tags := args.tags
```

Comment:
Why do we need the underscore here?

Reply:
Renamed because tags is the name of a package :laughing: